### PR TITLE
[Sequence.append(publisher)]

### PR DESCRIPTION
**Pull request by ngoccuong291**

### CODE REVIEW

File: CombineRx/Publishers/Sequence.swift [hunk 1]
```diff
@@ -73,6 +77,24 @@ extension Publishers.Sequence {
...
+//    public func prepend<S>(_ elements: S) -> Publishers.Sequence<Elements, Failure> where S : Sequence, Elements.Element == S.Element
+//
+//    public func prepend(_ publisher: Publishers.Sequence<Elements, Failure>) -> Publishers.Sequence<Elements, Failure>
+
+//    public func append(_ elements: Publishers.Sequence<Elements, Failure>.Output...) -> Publishers.Sequence<Elements, Failure>
+
+//    public func append<S>(_ elements: S) -> Publishers.Sequence<Elements, Failure> where S : Sequence, Elements.Element == S.Element
+//
+    public func append(_ publisher: Publishers.Sequence<Elements, Failure>) -> Publishers.Sequence<Elements, Failure> {
+
```
Comment (adamaszhu) [inline CombineRx/Publishers/Sequence.swift]: Can we not introduce this empty line style in this framework? Especially if there is just one line as the function body.

Comment (adamaszhu) [reply CombineRx/Publishers/Sequence.swift]: plus the redundant `self`

File: Sample/CombineRxSample/ViewController.swift [hunk 2]
```diff
@@ -29,5 +29,19 @@ class ViewController: UIViewController {
             receiveValue: { print($0) })
         publisher.subscribe(subscriber)
     }
+
```
Comment (adamaszhu) [inline Sample/CombineRxSample/ViewController.swift]: Can it be merged into `testSequence`?

File: Sample/Pods/CombineRx/CombineRx/Publishers/Sequence.swift [hunk 3]
```diff
@@ -25,6 +25,10 @@ extension Publishers {
         public init(sequence: Elements) {
             observable = Observable.from(sequence)
         }
+
+        private init(observable: Observable<Elements.Element>) {
+            self.observable = observable
+        }
```
Comment (adamaszhu) [inline Sample/Pods/CombineRx/CombineRx/Publishers/Sequence.swift]: I guess we need to git ignore `Pods` folder in the sample project.

File: CombineRx/Publishers/Sequence.swift (hunk 1 above)
Comment (ngoccuong291) [reply CombineRx/Publishers/Sequence.swift]: I'll remove the `self`, but the empty line really help me see things more clearly, without that, the code just blends into each other :(

**Pull request merged**